src/main/java/org/mule/extension/http/api/listener/server/HttpListenerConfig.java

```diff
@@ -51,9 +55,21 @@
   @Expression(NOT_SUPPORTED)
   private CorsInterceptorWrapper listenerInterceptors;
 
+  /**
+   * If true, request with an invalid value for "Transfer-Encoding" header will be rejected with a 400 Bad Request.
+   * @see InvalidTransferEncodingValidator
+   */
+  @Parameter
+  @Optional(defaultValue = "false")
+  @Expression(NOT_SUPPORTED)
+  private boolean rejectInvalidTransferEncoding;
```

Comment:
add `@since` for this as well. Des the sdk have some other way of indicationg this?

Reply:
SDK hasn't an annotation for this